src/freenas/usr/local/lib/middlewared_truenas/alert/source/license_status.py

```diff
@@ -189,15 +189,15 @@ def check_sync(self):
                             )
                         ))
 
-                # ES24F Expansion shelf
```

Comment:
@themylogin This is correct as  ES24F sorry for typo in original ticket. This is via Andrew Nyguen Product PM who caught my mistake in ticket